scripts/oldump.sh

```diff
@@ -102,8 +102,9 @@ ls -lhR
 function archive_dumps() {
     # copy stuff to archive.org
     # TODO: Switch to ia client tool. This will only work in production 'til then
-    python /olsystem/bin/uploaditem.py $dump --nowait --uploader=openlibrary@archive.org
-    python /olsystem/bin/uploaditem.py $cdump --nowait --uploader=openlibrary@archive.org
+    log "ia version is v$(ia --version)"  # ia version is v2.2.0
+    ia --config-file=/olsystem/etc/ia.ini upload $dump  $dump/  --metadata "collection:ol_exports" --metadata "year:${date:0:4}" --retries 300
```

Comment:
Let's set the `mediatype: data` (and `format`?)

Keep in mind that `uploader` is `openlibrary@archive.org` for production.

Reply:
I think that mediatype:data is the default, no need to set it.

Reply:
What should be the value of format?

Reply:
https://ia801402.us.archive.org/22/items/ol_cdump_2021-11-30/ol_cdump_2021-11-30_meta.xml

Reply:
format seems like a not standarded metadata field, so do we need to set it?

Reply:
will do it for consistancy..